test/tpmtest/tpmtest.c

```diff
@@ -8684,6 +8799,9 @@ void InitTpmTest()
 
     Tss2_Sys_Startup ( sysContext, TPM_SU_CLEAR );
     CheckTpmType();
+
+    GetCommands( sysContext, &supportedCommands );
```

Comment:
You return a TSS2_RC from GetCommands but you it's not checked here. This could result in the supportedCommands pointer being uninitialized when you go to check it later?